main.py

```diff
@@ -12,8 +12,19 @@
 from fastapi.openapi.docs import get_swagger_ui_html, get_swagger_ui_oauth2_redirect_html
 from fastapi.responses import HTMLResponse
 from pydantic import BaseModel
+from typing import Optional
 
 
+class CompilerErrorModel(BaseModel):
+    status: str = "failed"
+    message: str
+    column: Optional[int] = None
+    line: Optional[int] = None
```

Comment:
I think we are using 3.10 for this service and we don't need to use `Optional` anymore
```suggestion
    column: int | None = None
    line: int | None = None
```